### DISCUSSION TITLE
Postman POST not working: "Invalid form data: no form file found for extensions" error

### DISCUSSION THREAD

**Discussion by muhyilmaz** [gotenberg/gotenberg#500] [Q&A]
Hi,

I post my file path to Gotenberg API with Postman, it does not work.

Here is my headers:
![image](https://user-images.githubusercontent.com/34422587/191026008-42642e30-c567-4a73-a4ae-a38a73c13176.png)

Here is my form-data:
![image](https://user-images.githubusercontent.com/34422587/191026098-f61cb803-c716-452a-ac40-0d7b5f13ab8e.png)

Here is my gotenberg api works:
![image](https://user-images.githubusercontent.com/34422587/191026312-71b825bb-c577-4c78-b0eb-905229eaf983.png)

What is my mistake?

**Comment by gulien** [comment]
I think you're sending the file path and not the file itself.
